test/hmcda.jl/hmcda_geweke.jl

```diff
@@ -43,7 +43,8 @@ x = [s[:y][1]...]
 s_bk = Array{Turing.Chain}(undef, N)
 
 simple_logger = Base.CoreLogging.SimpleLogger(stderr, Base.CoreLogging.Debug)
-with_logger(simple_logger) do
+Base.CoreLogging.with_logger(simple_logger) do
```

Comment:
WIP ?

Reply:
Again was not working when I copied and pasted the code into REPL to check the new compiler works, so I made a quick fix! These should probably be separated out in a separate PR.